2021-09-04 17:59:05,868 INFO    MainThread:2580 [wandb_setup.py:_flush():69] setting env: {}
2021-09-04 17:59:05,869 INFO    MainThread:2580 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-04 17:59:05,869 INFO    MainThread:2580 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-04 17:59:05,870 INFO    MainThread:2580 [wandb_init.py:_log_setup():342] Logging user logs to F:\5th sem project\wandb\run-20210904_175905-u8occ330\logs\debug.log
2021-09-04 17:59:05,870 INFO    MainThread:2580 [wandb_init.py:_log_setup():343] Logging internal logs to F:\5th sem project\wandb\run-20210904_175905-u8occ330\logs\debug-internal.log
2021-09-04 17:59:05,872 INFO    MainThread:2580 [wandb_init.py:init():375] calling init triggers
2021-09-04 17:59:05,881 INFO    MainThread:2580 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-09-04 17:59:05,882 INFO    MainThread:2580 [wandb_init.py:init():395] re-initializing run, found existing run on stack: 3nm6yh53
2021-09-04 17:59:05,882 INFO    MainThread:2580 [wandb_run.py:finish():1271] finishing run khushal/final-weather-recognization/3nm6yh53
2021-09-04 17:59:05,883 INFO    MainThread:2580 [jupyter.py:save_history():353] not saving jupyter history
2021-09-04 17:59:05,883 INFO    MainThread:2580 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-09-04 17:59:05,883 INFO    MainThread:2580 [wandb_init.py:_jupyter_teardown():275] cleaning up jupyter logic
2021-09-04 17:59:05,884 INFO    MainThread:2580 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-09-04 17:59:05,884 INFO    MainThread:2580 [wandb_run.py:_restore():1543] restore
2021-09-04 17:59:06,081 INFO    MainThread:2580 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 476
}

2021-09-04 17:59:06,708 INFO    MainThread:2580 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 476
}

2021-09-04 17:59:07,583 INFO    MainThread:2580 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 476
}

2021-09-04 17:59:08,583 INFO    MainThread:2580 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19688
}

2021-09-04 17:59:08,684 INFO    MainThread:2580 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19688
}

2021-09-04 17:59:08,785 INFO    MainThread:2580 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19688
}

2021-09-04 17:59:08,887 INFO    MainThread:2580 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19688
}

2021-09-04 17:59:08,990 INFO    MainThread:2580 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19688
}

2021-09-04 17:59:09,093 INFO    MainThread:2580 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19688
}

2021-09-04 17:59:09,195 INFO    MainThread:2580 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19688
}

2021-09-04 17:59:09,297 INFO    MainThread:2580 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19688
}

2021-09-04 17:59:09,398 INFO    MainThread:2580 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19688
}

2021-09-04 17:59:09,500 INFO    MainThread:2580 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19688
}

2021-09-04 17:59:09,601 INFO    MainThread:2580 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6976
  total_bytes: 19688
}

2021-09-04 17:59:09,704 INFO    MainThread:2580 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19688
  total_bytes: 19688
}

2021-09-04 17:59:09,807 INFO    MainThread:2580 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19688
  total_bytes: 19688
}

2021-09-04 17:59:09,911 INFO    MainThread:2580 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19688
  total_bytes: 19688
}

2021-09-04 17:59:10,014 INFO    MainThread:2580 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19688
  total_bytes: 19688
}

2021-09-04 17:59:10,117 INFO    MainThread:2580 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19688
  total_bytes: 19688
}

2021-09-04 17:59:10,221 INFO    MainThread:2580 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19688
  total_bytes: 19688
}

2021-09-04 17:59:10,763 INFO    MainThread:2580 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19688
  total_bytes: 19688
}

2021-09-04 17:59:10,866 INFO    MainThread:2580 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19688
  total_bytes: 19688
}

2021-09-04 17:59:12,237 INFO    MainThread:2580 [wandb_run.py:_show_files():1915] logging synced files
2021-09-04 17:59:12,242 INFO    MainThread:2580 [wandb_init.py:init():424] starting backend
2021-09-04 17:59:12,243 INFO    MainThread:2580 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-09-04 17:59:12,248 INFO    MainThread:2580 [backend.py:ensure_launched():135] starting backend process...
2021-09-04 17:59:12,386 INFO    MainThread:2580 [backend.py:ensure_launched():139] started backend process with pid: 10684
2021-09-04 17:59:12,387 INFO    MainThread:2580 [wandb_init.py:init():429] backend started and connected
2021-09-04 17:59:12,387 INFO    MainThread:2580 [wandb_run.py:_label_probe_notebook():815] probe notebook
2021-09-04 17:59:12,388 INFO    MainThread:2580 [wandb_run.py:_label_probe_notebook():822] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-09-04 17:59:12,388 INFO    MainThread:2580 [wandb_init.py:init():477] updated telemetry
2021-09-04 17:59:12,389 INFO    MainThread:2580 [wandb_init.py:init():500] communicating current version
2021-09-04 17:59:16,203 INFO    MainThread:2580 [wandb_init.py:init():505] got version response 
2021-09-04 17:59:16,203 INFO    MainThread:2580 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-09-04 17:59:16,856 INFO    MainThread:2580 [wandb_init.py:init():540] starting run threads in backend
2021-09-04 17:59:21,873 INFO    MainThread:2580 [wandb_run.py:_console_start():1601] atexit reg
2021-09-04 17:59:21,876 INFO    MainThread:2580 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-09-04 17:59:21,877 INFO    MainThread:2580 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-09-04 17:59:21,883 INFO    MainThread:2580 [wandb_run.py:_redirect():1536] Redirects installed.
2021-09-04 17:59:21,884 INFO    MainThread:2580 [wandb_init.py:init():565] run started, returning control to user process
2021-09-04 17:59:43,048 INFO    MainThread:2580 [wandb_setup.py:_flush():69] setting env: {}
2021-09-04 17:59:43,048 INFO    MainThread:2580 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-04 17:59:43,048 INFO    MainThread:2580 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-04 17:59:43,049 INFO    MainThread:2580 [wandb_init.py:_log_setup():342] Logging user logs to F:\5th sem project\wandb\run-20210904_175943-r322r5co\logs\debug.log
2021-09-04 17:59:43,049 INFO    MainThread:2580 [wandb_init.py:_log_setup():343] Logging internal logs to F:\5th sem project\wandb\run-20210904_175943-r322r5co\logs\debug-internal.log
2021-09-04 17:59:43,050 INFO    MainThread:2580 [wandb_init.py:_jupyter_setup():294] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001D60653F730>
2021-09-04 17:59:43,071 INFO    MainThread:2580 [wandb_init.py:init():375] calling init triggers
2021-09-04 17:59:43,071 INFO    MainThread:2580 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-09-04 17:59:43,072 INFO    MainThread:2580 [wandb_init.py:init():395] re-initializing run, found existing run on stack: u8occ330
2021-09-04 17:59:43,072 INFO    MainThread:2580 [wandb_run.py:finish():1271] finishing run khushal/final-weather-recognization/u8occ330
